.gitignore

```diff
@@ -16,3 +16,6 @@
 /test/src/*.dylib
 /test/src/*.o
 /test/src/*.so
+
+# Do NOT ignore binaries placed in subdirs under '/test/bin'.
+!/test/bin/*/*
```

Comment:
I added it late last night without thinking, just realized that my global `.gitignore` was the source of my troubles. I'll undo that.